src/gui/editors/SongEditor.cpp

```diff
@@ -697,6 +705,7 @@ SongEditorWindow::SongEditorWindow(Song* song) :
 	zoomToolBar->addWidget( m_zoomingComboBox );
 
 	connect(song, SIGNAL(projectLoaded()), this, SLOT(adjustUiAfterProjectLoad()));
+	connect( this, SIGNAL(resized() ), m_editor, SLOT( updatePositionLine() ) );
```

Comment:
please correct the formatting on this, either have or have not spaces in the parenthesis, but this mix-and-match looks inconsistent.